src/client/features/search/index.js

```diff
@@ -125,16 +133,20 @@ class Search extends React.Component {
       const dsInfo =_.isEmpty(state.dataSources)? {iconUrl:null , name:''}: _.find(state.dataSources, ds => {
         return ds.uri === result.dataSource[0];
       });
+    
+    //get the link to source website from the map defined at start of file.
+    const sourceLink = sourceLinkMap[dsInfo.id] || "";
 
       return h('div.search-item', [
        h('div.search-item-icon',[
-          h('img', {src: dsInfo.iconUrl})
-        ]),
+          h('a',{href:sourceLink,target:"_blank"},[
```

Comment:
You should use the react router Link component.
https://github.com/PathwayCommons/app-ui/pull/745/files#diff-a1f9f779cabf8780553a2f0a90378e43L3
https://reacttraining.com/react-router/web/example/basic